content/02.introduction.md

```diff
@@ -15,14 +15,13 @@ The Pediatric Brain Tumor Atlas (PBTA) initiative established in 2018 by the Chi
 Just as cooperation accelerates specimens and data sharing, collaboration among researchers including computational and bench scientists, clinicians, and pathologists is critical for rigorous genomic analysis.
 
 Although there has been significant progress elucidating genomic bases of pediatric brain tumor formation and progression, translating therapeutic agents to phase II or III clinical trials and subsequent FDA approvals have not kept pace.
-Within the last 20 years, the FDA has approved only seven targeted agents for treating pediatric brain tumors: mTOR inhibitor everolimus, for subependymal giant cell astrocytoma; anti-PD-1 immunotherapy pembrolizumab, for microsatellite instability–high or mismatch repair–deficient tumors; NTRK inhibitors larotrectinib and entrectinib, for tumors with an NTRK 1/2/3 gene fusions; MEK1/2 inhibitor selumetinib, for neurofibromatosis type 1 (NF1) and symptomatic, inoperable plexiform neurofibromas; and combination therapy MEK1/2 inhibitor trametinib and BRAF/CRAF inhibitor dabrafenib for unresectable or metastatic progressive tumors with BRAF V600E mutations [@url:https://www.fda.gov/about-fda/oncology-center-excellence/pediatric-oncology-drug-approvals].
-
-This is, in part, due to pharmaceutical company priorities and toxicity concerns, posing challenges for researchers to obtain new therapeutic agents for pediatric clinical trials.
+Within the last 20 years, the FDA has approved only seven targeted agents for treating pediatric brain tumors [@url:https://www.fda.gov/about-fda/oncology-center-excellence/pediatric-oncology-drug-approvals].
```

Comment:
Instead of fully cutting these, what if we list the therapies and MOA without tumor types targeted? I hate to cut this, as it is _the impetus_ for all of our work.

Reply:
What, specifically, is the impetus? It was not clear to me from how was written.

Reply:
The lack of FDA approved drugs over decades which can be used for pediatric brain tumors, hence we urgently need to identify more novel therapeutics to target these tumors.

Reply:
> The lack of FDA approved drugs over decades which can be used for pediatric brain tumors

is covered by:

> Within the last 20 years, the FDA has approved only seven targeted agents for treating pediatric brain tumors 

In my opinion. 

We are in a position where we have to cut many words to meet our limit.